woocommerce/woocommerce-paypal-payments
# New Settings UI: Rename Dashboard to Overview (3944)

### Description

This PR renames the `Dashboard` tab (and all related strings) to `Overview`.

### Steps to test

1. Make sure the new settings still load correctly.
2. Ensure `Dashboard` got renamed to `Overview`.

### Screenshots

|Before|After|
|-|-|
|<img width="936" alt="dashboard" src="https://github.com/user-attachments/assets/6310efca-c25a-4303-9420-1c9691927e8b">|<img width="935" alt="overview" src="https://github.com/user-attachments/assets/d4d8818e-0526-423d-bbda-8850e6fcb9ee">|
